### PR TITLE
Add how to install for Amazon Linux

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: install/amazon-linux.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  * [Amazon Linux 2](#install-on-2)
+
+## How to install on Amazon Linux 2 {#install-on-2}
+
+You can use the following instruction to install PGroonga on Amazon Linux 2.
+
+Install `postgresql10` package:
+
+```text
```
Comment (kou) [inline install/amazon-linux.md]: We can use `console` here.

File: install/amazon-linux.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+% wget https://yum.postgresql.org/10/redhat/rhel-7-x86_64/postgresql10-server-10.9-1PGDG.rhel7.x86_64.rpm
```
Comment (kou) [inline install/amazon-linux.md]: PostgreSQL version should be variable.

See also:

  * https://github.com/pgroonga/pgroonga.github.io/blob/master/_config.yml#L15
  * https://github.com/pgroonga/pgroonga.github.io/blame/master/install/freebsd.md#L16

File: install/amazon-linux.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Here are supported Amazon Linux versions:
+
+  * [Amazon Linux 2](#install-on-2)
+
+## How to install on Amazon Linux 2 {#install-on-2}
+
+You can use the following instruction to install PGroonga on Amazon Linux 2.
+
+Install `postgresql10` package:
```
Comment (kou) [inline install/amazon-linux.md]: We need a description why we can't use PostgreSQL's Yum repository.

File: install/amazon-linux.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+Install `postgresql10-pgroonga` package:
+
+We enable EPEL repository as below.
+
+```text
+% sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+% sudo yum install -y epel-release
+% sudo yum-config-manager --enable epel
```
Comment (kou) [inline install/amazon-linux.md]: Is this needed?

File: install/amazon-linux.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+% sudo -H yum install -y postgresql10-*.rpm
+```
+
+Install `postgresql10-pgroonga` package:
+
+We enable EPEL repository as below.
+
+```text
+% sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+% sudo yum install -y epel-release
```
Comment (kou) [inline install/amazon-linux.md]: Is this needed?

File: install/amazon-linux.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+% sudo -H yum install -y postgresql10-*.rpm
+```
+
+Install `postgresql10-pgroonga` package:
+
+We enable EPEL repository as below.
+
+```text
+% sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```
Comment (kou) [inline install/amazon-linux.md]: `-H` is missing.

File: install/amazon-linux.md (hunk 4 above)
Comment (komainu8) [reply install/amazon-linux.md]: Ah...This is needless.
I've removed.

File: install/amazon-linux.md (hunk 5 above)
Comment (komainu8) [reply install/amazon-linux.md]: Ah...This is needless.
I've removed.

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

File: install/amazon-linux.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+You can use the following instruction to install PGroonga on Amazon Linux 2.
+
+Install `postgresql{{ site.amazon_linux_postgresql_version }}` package:
+
+We can't use PostgreSQL's Yum repository because of PostgreSQL doesn't provide a package for Amazon Linux.
+
+Therefore, we install PostgreSQL's RPM directly as below.
+
+```console
+% wget https://yum.postgresql.org/{{ site.amazon_linux_postgresql_version }}/redhat/rhel-7-x86_64/postgresql{{ site.amazon_linux_postgresql_version }}-server-{{ site.amazon_linux_postgresql_version }}.9-1PGDG.rhel7.x86_64.rpm
```
Comment (kou) [inline install/amazon-linux.md]: `10.9-1PGDG` should be variable too.

File: _config.yml [hunk 8]
```diff
@@ -13,6 +13,7 @@ windows_postgresql_versions:
   - 10
   - 11
 freebsd_postgresql_version: 11
+amazon_linux_postgresql_version: 10
```
Comment (kou) [inline _config.yml]:
```yaml
amazon_linux_postgresql_version:
  major: 10
  package: 10.9-1PGDG
```

**Comment by komainu8** [comment]
@kou Thank you for your comment.
I've fixed it.

File: install/amazon-linux.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+You can use the following instruction to install PGroonga on Amazon Linux 2.
+
+Install `postgresql{{ site.amazon_linux_postgresql_version.major }}` package:
+
+We can't use PostgreSQL's Yum repository because of PostgreSQL doesn't provide a package for Amazon Linux.
+
+Therefore, we modify PostgreSQL's `.repo` file as below.
+
+```console
+% wget https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
```
Comment (kou) [inline install/amazon-linux.md]: Can we use `yum` here?

Comment (kou) [reply install/amazon-linux.md]: Ah, we can't. Because we need to use `--nodeps` with `rpm`.

**Pull request merged**